### PR TITLE
fix Line3DOverlays bound updates

### PR DESCRIPTION
**Pull request by sethalves**
- fix a bug that kept the render-bounds of Line3DOverlays from being updated

### CODE REVIEW

**Comment by sethalves** [comment]
to test:
- activate edit.js
- create a cube
- grab the top-center affordance (to make the cube taller) and adjust it up or down
- make sure you see the vertical green line
- turn 180 degrees and do this again

**Comment by hifi-gustavo** [comment]
Build console output for commit: adc3581d63cad51d359e556d9f6e82b65991ada2 available at: https://gist.github.com/e8bbd9202e34a08e5d208b13d7ca9bbc<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9831-adc3581.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9831-adc3581.dmg

**Comment by sethalves** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: adc3581d63cad51d359e556d9f6e82b65991ada2 available at: https://gist.github.com/e8bbd9202e34a08e5d208b13d7ca9bbc<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9831-adc3581.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9831-adc3581.dmg

**Comment by claudioshg** [comment]
Testing

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9831-adc3581.exe

tl;dr:
- The cube doesn't move, stretch, or rotate using hand devices. 
Modifying the cube is possible but only if you use the mouse.
Using the mouse in Desktop Mode works perfect, but in VR the mouse produces many bugs.

In HMD using Hand Devices:
- Loading edit.js made the cube immovable. 
Creating a cube was possible. 
Then grabbing the top-center or any other affordance yielded no freedom to adjust the cube in any way.
Clicking on the rotation icon, and then trying to rotate the cube, generated a 0 degrees confirmation on the UI.
Grabbing on the top-center affordance did render a vertical green line, but I couldn't adjust the height.

In HMD using Mouse:
- Trying the same as above created even weirder results.
Grabbing the top-center affordance, created a secondary cube in the same spot as the original cube.
The tablet is frozen in it's spot and cannot be interacted with. However, I can toggle another tablet directly on top of the original.
Most importantly to know is that modifying the cube is now possible, but only by using the mouse. The hand devices don't work.

In Desktop Mode:
- Modifying the cube, as aimed in this PR, is possible in this mode.
Grabbing the top-center affordance and seeing the vertical green line is a success.
Making the cube taller is a success.
Rotating 180 degrees and then making the cube taller from the underside is also a success.

**Comment by sethalves** [comment]
@claudioshg I think most of these problems are fixed in another PR.  This one is all about that green line.  In the cases where you were able to use the affordance, did you always see the green line?

**Comment by claudioshg** [comment]
@sethalves Yes. In the case where I was able to use the top-center affordance, a vertical green light does appear.
In HMD with hand devices, the top-center affordance produced no vertical green light at all.
hifi-snap-by-claudioshg-on-2017-03-08_15-40-44
In HMD, using the mouse, did produce a verical green light.
hifi-snap-by-claudioshg-on-2017-03-08_15-41-14

**Comment by sethalves** [comment]
Okay.  Let's consider this a QA pass, then.  I believe the other problems you experienced are mostly sorted out in the port of edit.js to the tablet.

**Comment by claudioshg** [comment]
Roger that @sethalves
